depscan/lib/analysis.py

```diff
@@ -1465,7 +1481,7 @@ def process_vuln_occ(bom_dependency_tree, direct_purls, oci_product_types, optio
             }
         elif vid.startswith("GHSA") or vid.startswith("npm"):
             source = {
-                "name": "GitHub Advisory",
+                "name": "GitHub",
```

Comment:
GitHub advisory sounds better here

Reply:
@prabhu I think I changed that because it may not be a GitHub Advisory ID but an NPM one. But I can change it back if you prefer.

Reply:
Since npm is also owned by GitHub, we can call both `GitHub Advisory` and use `GitHub` when referring to the repo or PR.